pkgs/tools/text/ocrmypdf/default.nix

```diff
@@ -77,23 +79,7 @@ buildPythonApplication rec {
   ];
 
   # The tests take potentially 20+ minutes, depending on machine
-  doCheck = false;
-
-  # These tests fail and it might be upstream problem... or packaging. :)
-  # development is happening on macos and the pinned test versions are
-  # significantly newer than nixpkgs has. Program still works...
-  # (to the extent I've used it) -- Kiwi
-  checkPhase = ''
-    export HOME=$TMPDIR
-    pytest -k 'not test_force_ocr_on_pdf_with_no_images \
-    and not test_tesseract_crash \
-    and not test_tesseract_crash_autorotate \
-    and not test_ghostscript_pdfa_failure \
-    and not test_gs_render_failure \
-    and not test_gs_raster_failure \
-    and not test_bad_utf8 \
-    and not test_old_unpaper'
-  '';
+  #doCheck = false;
```

Comment:
can this line be removed?

Reply:
I left it in because of the comment.

Reply:
I had the exact same thought proces as you two just had.

Reply:
Yeah, let's remove this line. If this turns out to be a problem, having to write this line is okay :-)

Reply:
Ok removed